Client/Application/AppDelegate.swift

```diff
@@ -78,6 +83,32 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         ReaderModeHandlers.register(server)
         server.start()
     }
+
+    private func setUserAgent() {
+        let webView = UIWebView()
+        let userAgent = webView.stringByEvaluatingJavaScriptFromString("navigator.userAgent")!
```

Comment:
I am not sure if we should look at the `UIWebView`. Are we 100% sure that it's user agent string will always match that of the `WKWebView`? I would not count on it.